tests/conftest.py

```diff
@@ -50,7 +50,7 @@ def mock_beamline_module_filepaths(bl_name, bl_module):
 
 
 @pytest.fixture(scope="function")
-def module_and_devices_for_beamline(request):
+def mappings(request):
```

Comment:
revert this rename it removes clarity and isn't mapped to the inputs of the tests anymore?

Reply:
reverted